experimental/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -36,6 +36,10 @@ export interface TinyliciousResources {
     containerServices: TinyliciousContainerServices;
 }
 
+export interface TinyliciousDetachedResources extends TinyliciousResources {
+    attach: () => void;
```

Comment:
This should be async.

```suggestion
    attach: async () => Promise<void>;
```

Reply:
FYI, i looked it up and on interfaces you don't need async keyword, just `Promise<void>`